src/main/kotlin/racingcar/domain/turn/TurnManager.kt

```diff
@@ -1,19 +1,19 @@
-package racingcar.turn
+package racingcar.domain.turn
 
-import racingcar.racing.car.Car
-import racingcar.racing.result.RacingResult
-import racingcar.racing.rule.CarMovementRule
+import racingcar.domain.car.Car
+import racingcar.domain.result.RacingResult
+import racingcar.domain.rule.CarMovementRule
 
-class SimpleTurnManager(
-    override val totalSteps: Int,
+class TurnManager(
+    val totalSteps: Int,
     private val cars: List<Car>,
     private val rule: CarMovementRule
-) : TurnManager {
+) {
 
     private var currentTurn = 0
     private val racingResult = arrayListOf<RacingResult>()
```

Comment:
코틀린은 읽기 전용 컬렉션과 변경 가능한 컬렉션을 구별해 제공합니다. 아래의 코드로 개선해 보면 어떨까요?

```suggestion
    private val racingResult: MutableList<RacingResult> = mutableListOf()
```